inst/automation/git_urls.rb

```diff
@@ -65,7 +65,7 @@ def gh_default_branch(owner_repo)
 
 # add other repos (those not in ropensci or ropenscilabs)
 nms = ["package", "url"]
-url = 'https://raw.githubusercontent.com/ropenscilabs/makeregistry/master/inst/automation/not_transferred.txt'
+url = 'https://raw.githubusercontent.com/ropensci-org/makeregistry/master/inst/automation/not_transferred.txt'
```

Comment:
why is this read from GitHub and not from the source?

Reply:
just laziness i think